src/clr-icons/clr-icons-element.ts

```diff
@@ -57,6 +57,10 @@ ClarityIconElement.prototype._setIconSize = function(size: string) {
   }
 };
 
+ClarityIconElement.prototype._normalizeShape = function(value: string): string {
+  return value.split(/\s/)[0].toLowerCase();
+};
```

Comment:
It doesn't sit well with me that people can define shape as `home`, `HOME`, `HoMe`, or `HomE improvement` and all get the same icon. That's weird and not something you would expect from an API. I can't, for example, expect to use `@Component`, `@component`, and  `@ComPoneNT` in Angular.

But I acknowledge that we were already doctoring the shape string and reluctantly approve. As long as this is something that is only there to guard users against screwing up and not a feature we are promoting.

Reply:
Well, semantically HTML tags and attributes are not case sensitive, but values are. This is a departure from that, but I think most browsers handle this except in cases like `placeholder=""` where it is taken as case sensitive.